### PR TITLE
Dummy commit for testing Mergify

### PR DESCRIPTION
**Pull request by hugovk**

### CODE REVIEW

**Comment by hugovk** [comment]
@radarhere Please could you approve this?

At https://dashboard.mergify.io/installation/10048014/simulator?repositoryName=docker-images&pullRequestNumber=3, the simulator says:

---

#### Rule: Automatic merge (merge)
- [ ] `#approved-reviews-by>=1`
- [X] `label=automerge`
- [X] `status-success=continuous-integration/travis-ci/pr`

<hr />
:sparkling_heart:&nbsp;&nbsp;Mergify is proud to provide this service for free to open source projects.

:rocket:&nbsp;&nbsp;You can help us by [becoming a sponsor](/sponsors/Mergifyio)!
<hr />

**Review by radarhere — APPROVED** [review]

**Comment by hugovk** [comment]
Thanks, still not automerged. It now says:

---

#### Rule: Automatic merge (merge)
- [ ] `#approved-reviews-by>=1`
- [X] `label=automerge`
- [X] `status-success=continuous-integration/travis-ci/pr`

<hr />
:sparkling_heart:&nbsp;&nbsp;Mergify is proud to provide this service for free to open source projects.

:rocket:&nbsp;&nbsp;You can help us by [becoming a sponsor](/sponsors/Mergifyio)!
<hr />

The docs for `approved-reviews-by` says:

> This only matches reviewers with admin or write permission on the repository.

https://doc.mergify.io/conditions.html

I've invited you to this repo to see if that makes a difference. It could be that we just skip `approved-reviews-by` as a hard rule, and go by the `automerge` label (and passing CI). (I think it'd still be a good idea to approve when labelling, for clarity.) Only those with merge permission can add labels, so it's still safe.

**Comment by hugovk** [comment]
No longer needed, re: https://github.com/python-pillow/docker-images/pull/85#issuecomment-695799315

**Pull request closed without merging**
